src/Files.App/Helpers/InteropHelpers.cs

```diff
@@ -3,59 +3,60 @@
 
 using Microsoft.UI.Input;
 using Microsoft.UI.Xaml;
-using System;
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-namespace UWPToWinAppSDKUpgradeHelpers
+namespace Files.App.Helpers
 {
-	[ComImport]
-	[Guid("3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8")]
-	[InterfaceType(
-		ComInterfaceType.InterfaceIsIUnknown)]
-	interface IDataTransferManagerInterop
-	{
-		IntPtr GetForWindow([In] IntPtr appWindow, [In] ref Guid riid);
-
-		void ShowShareUIForWindow(IntPtr appWindow);
-	}
-
 	public static class InteropHelpers
 	{
 		public static readonly Guid DataTransferManagerInteropIID =
-			new Guid(0xa5caee9b, 0x8708, 0x49d1, 0x8d, 0x36, 0x67, 0xd2, 0x5a, 0x8d, 0xa0, 0x0c);
+			new(0xa5caee9b, 0x8708, 0x49d1, 0x8d, 0x36, 0x67, 0xd2, 0x5a, 0x8d, 0xa0, 0x0c);
```

Comment:
Let's keep `Guid`

Reply:
That can be simplified. however?

Reply:
We usually only simplify it if it's already clear what the type is. I just noticed that it's defined on line 13 so disregard what I said.